CI Flake: 'Setup Gradle' (Windows)

Seen this a few times, I believe @criticalAY had this locally after a gradle upgrade

https://github.com/ankidroid/Anki-Android/actions/runs/9255048898/job/25458050475

```
2024-05-27T12:44:32.1140353Z ##[group]Run gradle/actions/setup-gradle@v3
2024-05-27T12:44:32.1140761Z with:
2024-05-27T12:44:32.1140976Z   cache-read-only: false
2024-05-27T12:44:32.1141242Z   cache-disabled: false
2024-05-27T12:44:32.1141509Z   cache-write-only: false
2024-05-27T12:44:32.1141793Z   cache-overwrite-existing: false
2024-05-27T12:44:32.1142166Z   gradle-home-cache-includes: caches
notifications

2024-05-27T12:44:32.1142565Z   gradle-home-cache-cleanup: false
2024-05-27T12:44:32.1142861Z   add-job-summary: always
2024-05-27T12:44:32.1143148Z   add-job-summary-as-pr-comment: never
2024-05-27T12:44:32.1143475Z   dependency-graph: disabled
2024-05-27T12:44:32.1143799Z   dependency-graph-continue-on-failure: true
2024-05-27T12:44:32.1144147Z   build-scan-publish: false
2024-05-27T12:44:32.1144423Z   validate-wrappers: false
2024-05-27T12:44:32.1144692Z   generate-job-summary: true
2024-05-27T12:44:32.1144985Z   gradle-home-cache-strict-match: false
2024-05-27T12:44:32.1145533Z   workflow-job-context: {
  "os": "windows-latest",
  "iteration": 1,
  "name": "windows"
}
2024-05-27T12:44:32.1146196Z   github-token: ***
2024-05-27T12:44:32.1146412Z env:
2024-05-27T12:44:32.1146676Z   CODECOV_TOKEN: ***
2024-05-27T12:44:32.1147100Z   JAVA_HOME: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\21.0.3-9.0\x64
2024-05-27T12:44:32.1147766Z   JAVA_HOME_21_X64: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\21.0.3-9.0\x64
2024-05-27T12:44:32.1148254Z ##[endgroup]
2024-05-27T12:44:38.4762636Z Merged default JDK locations into C:\Users\runneradmin\.m2\toolchains.xml
2024-05-27T12:44:38.4769621Z ##[group]Restore Gradle state from cache
2024-05-27T12:44:40.6197838Z Received 209715200 of 772821052 (27.1%), 200.0 MBs/sec
2024-05-27T12:44:41.6260821Z Received 461373440 of 772821052 (59.7%), 219.6 MBs/sec
2024-05-27T12:44:42.6288178Z Received 713031680 of 772821052 (92.3%), 226.2 MBs/sec
2024-05-27T12:44:42.9110615Z Cache Size: ~737 MB (772821052 B)
2024-05-27T12:44:42.9149119Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/3cb8a6a6-93bf-4f14-830c-bd52a946e2bc/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:44:43.6301559Z Received 772821052 of 772821052 (100.0%), 183.7 MBs/sec
2024-05-27T12:45:11.5403690Z Cache restored successfully
2024-05-27T12:45:11.5758408Z Restored Gradle User Home from cache key: gradle-home-v1|Windows|unit[9458aee62b2c0ce7d26523702c1da1be]-fabbd7a3a599bb11b74f91205a2967828b893000
2024-05-27T12:45:12.1466061Z Cache Size: ~0 MB (241809 B)
2024-05-27T12:45:12.1466798Z Cache Size: ~1 MB (1005985 B)
2024-05-27T12:45:12.1467400Z Cache Size: ~0 MB (126709 B)
2024-05-27T12:45:12.3079378Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/72c1da32-c1fb-48ef-b5f3-35d39a34bad4/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:12.3105636Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/7c100505-e21c-4763-a50d-c2b11d11eadb/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:12.3127653Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/62aa60dc-64e4-4af0-a360-04e06014d544/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:12.6388685Z Cache restored successfully
2024-05-27T12:45:12.6503888Z Restored instrumented-jars with key gradle-instrumented-jars-v1-cfd7d89155a92ce6d1c5d5267d1cbb2e to C:\Users\runneradmin\.gradle\caches\jars-*\*/
2024-05-27T12:45:12.7336832Z Cache Size: ~36 MB (38112471 B)
2024-05-27T12:45:12.8692980Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/5c6fcb13-ac36-4aee-8ffb-b2ffa65ba437/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:12.9136359Z Received 241809 of 241809 (100.0%), 0.2 MBs/sec
2024-05-27T12:45:12.9138146Z Received 71303168 of 133065448 (53.6%), 67.8 MBs/sec
2024-05-27T12:45:12.9385816Z Received 1005985 of 1005985 (100.0%), 0.9 MBs/sec
2024-05-27T12:45:12.9386716Z Received 126709 of 126709 (100.0%), 0.1 MBs/sec
2024-05-27T12:45:12.9387331Z Received 38112471 of 38112471 (100.0%), 36.0 MBs/sec
2024-05-27T12:45:12.9387827Z Received 75497472 of 1507987020 (5.0%), 72.0 MBs/sec
2024-05-27T12:45:13.4052882Z Cache Size: ~127 MB (133065448 B)
2024-05-27T12:45:13.4504700Z Received 88080384 of 534779083 (16.5%), 83.2 MBs/sec
2024-05-27T12:45:13.4804380Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/cf4c541e-4645-44af-92cf-2f0facb4b85b/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:13.5046722Z Cache restored successfully
2024-05-27T12:45:13.5247040Z Restored generated-gradle-jars with key gradle-generated-gradle-jars-v1-d6d47bedd9337a0be6f3a426277e63ee to C:\Users\runneradmin\.gradle\caches\8.7\generated-gradle-jars\gradle-api-8.7.jar
2024-05-27T12:45:13.9223797Z Received 133065448 of 133065448 (100.0%), 63.3 MBs/sec
2024-05-27T12:45:13.9362515Z Received 243269632 of 1507987020 (16.1%), 115.7 MBs/sec
2024-05-27T12:45:14.4500025Z Received 209715200 of 534779083 (39.2%), 99.6 MBs/sec
2024-05-27T12:45:14.9388255Z Received 373293056 of 1507987020 (24.8%), 118.4 MBs/sec
2024-05-27T12:45:15.4577229Z Received 381681664 of 534779083 (71.4%), 120.7 MBs/sec
2024-05-27T12:45:15.9453288Z Received 574619648 of 1507987020 (38.1%), 136.5 MBs/sec
2024-05-27T12:45:16.2406607Z Cache Size: ~510 MB (534779083 B)
2024-05-27T12:45:16.2757776Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/89eaf7c2-0f66-4248-9fb0-63a63c83a723/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:16.4629667Z Received 534779083 of 534779083 (100.0%), 126.8 MBs/sec
2024-05-27T12:45:16.9456651Z Received 864026624 of 1507987020 (57.3%), 164.3 MBs/sec
2024-05-27T12:45:17.9481224Z Received 1212153856 of 1507987020 (80.4%), 192.2 MBs/sec
2024-05-27T12:45:18.8619975Z Cache Size: ~1438 MB (1507987020 B)
2024-05-27T12:45:18.8704021Z [command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/fc480c3e-9e4b-4b6f-85cd-3e99b8c49085/cache.tzst -P -C D:/a/Anki-Android/Anki-Android --force-local --use-compress-program "zstd -d"
2024-05-27T12:45:18.9581816Z Received 1507987020 of 1507987020 (100.0%), 204.7 MBs/sec
2024-05-27T12:45:21.8618054Z Cache restored successfully
2024-05-27T12:45:21.8682692Z Restored wrapper-zips with key gradle-wrapper-zips-v1-09242415bccaaeb0f8be2fc74b9ad33a to C:\Users\runneradmin\.gradle\wrapper\dists\gradle-8.7-bin\bhs2wmbdwecv87pi65oeuq5iu
2024-05-27T12:45:22.7042828Z Cache restored successfully
2024-05-27T12:45:22.7046384Z Restored groovy-dsl with key gradle-groovy-dsl-v1-97da7e10134d4298944637af11ea8c4d to C:\Users\runneradmin\.gradle\caches\*\groovy-dsl\*/
2024-05-27T12:45:48.2368489Z Cache restored successfully
2024-05-27T12:45:48.3102893Z Restored dependencies with key gradle-dependencies-v1-2b93be8353b68a27926cea5a8103e0c0 to C:\Users\runneradmin\.gradle\caches\modules-*\files-*\*\*\*\*
2024-05-27T12:46:52.6179586Z Cache restored successfully
2024-05-27T12:46:52.6184234Z Restored kotlin-dsl with key gradle-kotlin-dsl-v1-2f648f8e5a9ff86e70fe055b39ea37e2 to C:\Users\runneradmin\.gradle\caches\*\kotlin-dsl\accessors\*/
2024-05-27T12:46:52.6185356Z C:\Users\runneradmin\.gradle\caches\*\kotlin-dsl\scripts\*/
2024-05-27T12:49:32.1254823Z ##[error]The action 'Setup Gradle' has timed out after 5 minutes.
```

--- a/.github/workflows/tests_unit.yml
+++ b/.github/workflows/tests_unit.yml
@@ -111,7 +111,7 @@ jobs:
     # Do not run the scheduled jobs on forks
     if: (github.event_name == 'schedule' && github.repository == 'ankidroid/Anki-Android') || (github.event_name != 'schedule')
     needs: matrix_prep
-    timeout-minutes: 30
+    timeout-minutes: 35
     env:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
     strategy:
@@ -141,7 +141,7 @@ jobs:
 
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v3
-        timeout-minutes: 5
+        timeout-minutes: 10
         with:
           # Only write to the cache for builds on the 'main' branches, stops branches evicting main cache
           # Builds on other branches will only read from main branch cache writes

